README.md

```diff
@@ -44,6 +44,82 @@ cd grofer
 go build grofer.go
 ```
 
+### Docker
+
+1. Get docker image using
+   ```
+   TODO: docker image is not uploaded yet
```

Comment:
Now I guess we can fill this in 🎉

Reply:
Yes, I'm doing that rn.

I can't exactly test it since `latest` isn't pushed yet. Pulling using `pr-124` tag works though.